package/libs/toolchain/Makefile

```diff
@@ -7,7 +7,7 @@
 
 include $(TOPDIR)/rules.mk
 PKG_NAME:=toolchain
-PKG_RELEASE:=5
+PKG_RELEASE:=6
```

Comment:
I would not change this for the glibc update. such updates should be done by a full sysupgrade.